microsetta_public_api/api/diversity/alpha.py

```diff
@@ -1,10 +1,23 @@
 from flask import jsonify
+from microsetta_public_api.models._alpha import Alpha
+from microsetta_public_api.models._exceptions import UnknownID
+from microsetta_public_api.repo._alpha_repo import AlphaRepo
 
 
 def get_alpha(sample_id, alpha_metric):
+    alpha_repo = AlphaRepo()
+    try:
```

Comment:
I don't think it's necessarily exceptional behavior for a sample to not exist. What about something like:

```
if not alpha_repo.exists(sample_id):
    return jsonify(...)

alpha_series = ...
```

And since this looks like its getting lined up for multi-sample, this could extend to:

```python
if not all(alpha_repo.exists([sample_ids])):
    return jsonify(error=404, ...")
```

Reply:
Sure. I removed the `UnknownID` error, changed the docstring of `get_alpha_diversity` so that it returns the union of `sample_ids` and known id's, and added a place for an `exists` method.
https://github.com/biocore/microsetta-public-api/pull/3/commits/348faa651ab20212da08614f5100b884fa7e9222